metals/src/main/scala/scala/meta/internal/metals/FoldingRangeProvider.scala

```diff
@@ -0,0 +1,65 @@
+package scala.meta.internal.metals
+
+import java.util
+
+import org.eclipse.lsp4j.{FoldingRange, FoldingRangeCapabilities}
+
+import scala.meta.Term.Block
```

Comment:
Nit, by convention we `import scala.meta._` and reference classes like `Term.Block`. This is because many names of the inner classes conflict, for example `Term.Name` and `Type.Name`